.vscode/settings.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
Do we need this?

Reply:
I removed this in the most recent commit! it was an accidental file commited